### PR TITLE
Kubernetes tooltips

### LINKED ISSUE #587: Make the tooltip smart

### PR DESCRIPTION
**Pull request by jkfran**
## Done

Make the tooltip smart

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check a k8s tooltip: https://charmhub-io-599.demos.haus/ambassador
- Check a regular tooltip: https://charmhub-io-599.demos.haus/jenkins

## Issue / Card

Fixes #587

### CODE REVIEW

File: templates/details/_details-header.html [hunk 1]
```diff
@@ -73,7 +73,13 @@ <h1 class="p-media-object__title">
             </div>
             <div class="p-tooltip__modal" aria-controls="icon-tooltip-modal" id="icon-tooltip-modal">
               <div class="p-tooltip__dialog" role="dialog" aria-labelledby="modal-content">
-                <p id="modal-content" class="u-no-margin--bottom u-no-padding--top">The deployment of charms is done using Juju, our Universal Operator Lifecycle Manager. <a href="https://juju.is/docs/microk8s-cloud">Learn how to deploy a charm using MicroK8s</a>.</p>
+                <p id="modal-content" class="u-no-margin--bottom u-no-padding--top">
+                  {% if "kubernetes" in package.store_front.os %}
+                    k8s.
```
Comment (jkfran) [inline templates/details/_details-header.html:78]: @tbille 👀

**Comment by clagom** [comment]
Copydoc: https://docs.google.com/document/d/112jQ8Jy-Egnymuc7PlZ52sV_FznAIYqQ22_gsyoQYko/edit

**Comment by danielmutis** [comment]
In the tooltip, the margin should be 1rem. The first line of text should be aligned with the font on the command. 

Screenshot 2020-11-12 at 10 03 00

Screenshot 2020-11-12 at 10 03 53

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**